.github/ISSUE_TEMPLATE/bug-report.md

```diff
@@ -0,0 +1,50 @@
+---
+name: Bug report
+about: Unexpected behavior? Tell us more
+title: ''
+labels: bug
+assignees: ''
+
+---
+
+**Self checklist**
+
+ - [ ] This issue hasn't been already created
+ - [ ] Everything is sound and clear, all sections are filled-up
+ - [ ] Relevant labels have been added
+
+# Expected Behavior (short explanation)
+
+Please describe the behavior you are expecting
+
+# Current Behavior
+
+What is the current behavior?
+
+# Failure Information
```

Comment:
How is current behavior different from Failure Information
?

Reply:
Failure information is just a section that contains the *steps to reproduce*, the *context* and the *failure logs,* whereas the *current behavior* is asking for explanation about what is going on.  Perhaps this is not clear

Reply:
Is it more clear now? https://github.com/inmanta/inmanta/pull/2416/commits/5090b071922f322cee876f9d6288b7279514d3b2